platform/lib/platform/material/media.ex

```diff
@@ -163,6 +169,16 @@ defmodule Platform.Material.Media do
     end)
   end
 
+  defp populate_geolocation(media) do
```

Comment:
Can we rename `media` to `changeset` because the value passed to this function is not media, but rather a changeset?

Reply:
Done!